bazaarvoice/emodb
# Update Consistency to CL_TWO in both US and EU region.

Jira Issue
[[PD-XXXX](https://bazaarvoice.atlassian.net/browse/PD-XXXX)](https://bazaarvoice.atlassian.net/browse/PD-236935)

What Are We Doing Here?
Updating Read Consistency to LOCAL_QUORUM resulted in Databus load average being spiked in both EU and US region.

How to Test and Verify
mvn clean install
Verify the SOR, Databus and Queue APIs

![image](https://github.com/bazaarvoice/emodb/assets/134580247/b002612d-ca6e-4be3-8123-611b923d880f)


Risk
Level
Low
